### PR TITLE
feat: Allow for setting the excludedPathsList key to an array of paths (strings) that can be appended to the urlExclusionsList property.

### LINKED ISSUE #11: [Feature Request] Allow for setting additional URLs to exclude via profile (UserDefaults)
**Is your feature request related to a problem? Please describe.**
The exclusions list could be appended to without needing to rebuild the project. This can be done easily with `UserDefaults` as there are already options configured like this already.

**Describe the solution you'd like**

1. Rename existing `urlExclusionsList` to `defaultExclusionsList`
2. Introduce a new `excludedPathsListKey` property to define the key we will use
3. Create a new `urlExclusionsList` static var that combines the `defaultExclusionsList` and managed exclusions via `UserDefaults` by mapping the new `excludedPathsList` key to an array of `URL`s

### PR DESCRIPTION
**Pull request by ryangball**
…s (strings) that can be appended to the urlExclusionsList property.

*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*
Close #11 

## Pre-launch Checklist

- [X ] I read the [Code of Conduct](/CODE_OF_CONDUCT.md) and followed the process outlined there for submitting PRs.
- [ X] I listed at least one issue that this PR fixes in the description above.
- [X ] I updated/added relevant documentation (doc comments with `///`).
- [ X] I updated [README.md](https://github.com/IBM/mac-ibm-migration-tool/blob/main/README.md) file with new version/info - if applicable.
- [NA ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [NA] All existing and new tests are passing.

### CODE REVIEW

File: migrator/AppContext.swift [hunk 1]
```diff
@@ -105,11 +108,16 @@ struct AppContext {
     static var duplicateFilesHandlingPolice: DuplicateFilesHandlingPolicy {
         return DuplicateFilesHandlingPolicy(rawValue: UserDefaults.standard.string(forKey: Self.duplicateFilesHandlingPolicyKey) ?? "overwrite") ?? .overwrite
     }
-    
+    static var urlExclusionsList: [URL?] {
```
Comment (SMartorelli) [inline migrator/AppContext.swift:111]: The previous name was `urlExclusionList` if you want to rename it `urlExclusionsList` you must change the reference to this variable in the other files of the project.

**Review by SMartorelli — CHANGES_REQUESTED** [review]
Thanks for making this PR!
Please check the comment about `urlExclusionList` variable name.

File: migrator/AppContext.swift (hunk 1 above)
Comment (ryangball) [reply migrator/AppContext.swift]: Sorry @SMartorelli, this has been resolved with 76e635f3aabe016f27f36e4bbd13482c031cb380.

Comment (ryangball) [reply migrator/AppContext.swift]: Make that 2e308f95ff59e4f7322fbaee45aa842c200ae99d due to signing commit.

**Review by SMartorelli — APPROVED** [review]
LGTM

**Pull request merged**